glean-core/src/histogram/functional.rs

```diff
@@ -66,7 +70,11 @@ impl Histogram<Functional> {
     }
 
     /// Get a snapshot of all contiguous values.
-    pub fn snapshot_values(&self) -> HashMap<u64, u64> {
+    ///
+    /// **Caution** This is a more specific implemenation of `snapshot_values` on functional
```

Comment:
```suggestion
    /// **Caution** This is a more specific implementation of `snapshot` on functional
```

Reply:
I keep typing the same mistake in ~~implemenation~~ implementation.

`snapshot_values` is correct though. That's the function that the other histograms use. We can't overwrite that, so we have to use a new function name, which is why I also added that comment